tests/unit/api/BackbeatAPI.spec.js

```diff
@@ -5,15 +5,18 @@ const BackbeatRequest = require('../../../lib/api/BackbeatRequest');
 const config = require('../../../lib/Config');
 const fakeLogger = require('../../utils/fakeLogger');
 const setupIngestionSiteMock = require('../../utils/mockIngestionSite');
+const locationConfig = require('../../../conf/locationConfig.json');
 
 describe('BackbeatAPI', () => {
     let bbapi;
 
     const destconfig = config.extensions.replication.destination;
     const { site } = destconfig.bootstrapList[0];
     const ingestSite = `${destconfig.bootstrapList[1].site}-ingestion`;
+    const coldSite = Object.keys(locationConfig)
+        .filter(site => locationConfig[site].isCold)[0];
 
-    before(() => {
+    beforeEach(() => {
```

Comment:
Do we really need to create a new `BackbeatAPI` before every test ? Why wasn't it like that before ?